### PR TITLE
[2.7] Enable automated install of vsphere cpi/csi

### PR DESCRIPTION
**Pull request by slickwarren**
port of https://github.com/rancher/shepherd/pull/103

### CODE REVIEW

**Comment by caliskanugur** [comment]
Could we get a rebase for this one?

File: extensions/rke1/nodetemplates/vsphere/create.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ const vmwarevsphereNodeTemplateNameBase = "vmwarevsphereNodeConfig"
 // an VSphere node template and returns the NodeTemplate response
```
Comment (caliskanugur) [inline extensions/rke1/nodetemplates/vsphere/create.go:14]: There's a diff, here cause of `MergeOverride`.  It seems good to me, should be the cause of 2.7 vs 2.8 & 2.9

**Review by caliskanugur — APPROVED** [review]
LGTM

**Review by igomez06 — APPROVED** [review]
LGTM

**Pull request merged**